examples/weight-width.sil

```diff
@@ -0,0 +1,13 @@
+\begin[class=book,papersize=129mm x 198mm]{document}
+
+\font[family=DejaVu Sans]{
```

Comment:
I'm okay with this if we don't have a better choice in our current test repertoire already. I wouldn't add a new font to the test downloads for this, but given DejaVu is unmaintained and starting to age badly I'd prefer something else if we have one with widths already.

Also we should include a sample using a variable font in this test.

Reply:
Is that actually required? We don't support variable fonts at all unless they have instances — right? Or even then we don't I thought.